readthedocs/templates/projects/domain_form.html

```diff
@@ -0,0 +1,53 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Edit Domains" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-proj %}class="active"{% endblock %}
+
+{% block project-domains-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Domains" %}{% endblock %}
+
+{% block project_edit_content %}
+  <h3> {% trans "Existing Domains" %} </h3>
+  <p>
+    <ul>
+    {% for relationship in domains %}
+        <li>
+        <a href="{{ relationship.get_absolute_url }}">
+        {{ relationship.child }}
+        </a>
+        (<a href="{% url "projects_domains_delete" relationship.parent.slug relationship.child.slug %}">{% trans "Remove" %}</a>)
+        </li>
+    {% endfor %}
+    </ul>
+  <p>
+    {% trans "Choose which project you would like to add as a domain." %}
+  </p>
+  {% if domain %}
+  <form method="post" action="{% url 'projects_domains_edit' project.slug domain.pk %}">{% csrf_token %}
+  {% else %}
+  <form method="post" action="{% url 'projects_domains_create' project.slug %}">{% csrf_token %}
+  {% endif %}
```

Comment:
Such a case could be easier written with:

```
{% if domain %}
    {% url 'projects_domains_create' project.slug as action_url %}
{% else %}
    {% url 'projects_domains_edit' project.slug domain.pk as action_url %}
{% endif %}
<form method="post" action="{{ action_url }}">{% csrf_token %}
```

It avoids code duplication and is easier to read. Plus it won't confuse HTML
editors about which of the elements matches the single closing `</form>` :)